### ISSUE TITLE
[BUG] Unable to disable individual rules

### ISSUE THREAD

**Issue by NickBalnaves** [dart-code-checker/dart-code-metrics#752]
* **Dart code metrics version:** 4.13.0
* **Dart sdk version:** 2.16.1

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```yaml
include: ../analysis_options.yaml
dart_code_metrics:
  rules:
    no-empty-block: false
```

</details>

**What did you expect to happen?**
Passing in false should disable an individual rule as it does for the Flutter analyzer https://dart.dev/guides/language/analysis-options#disabling-individual-rules

**What actually happened?**
All dart_code_metrics rules were disabled after trying to disable one rule

**Are you willing to submit a pull request to fix this bug?**
No

**Comment by dkrutskikh** [comment]
@NickBalnaves can you provide repository with minimal representative project?

**Comment by NickBalnaves** [comment]
False alarm
